src/org/exist/xquery/parser/XQuery.g

```diff
@@ -1884,8 +1886,10 @@ reservedKeywords returns [String name]
 	|
 	"variable" { name= "variable"; }
 	|
-	"namespace" { name= "namespace"; }
+	"namespace-node" { name= "namespace-node"; }
 	|
+    "namespace" { name= "namespace"; }
```

Comment:
I used the same spacing convention as used in the rest of this Antlr grammar file.